package.json

```diff
@@ -104,7 +104,7 @@
     "regl": "^1.3.11",
     "regl-error2d": "^2.0.6",
     "regl-line2d": "3.0.13",
-    "regl-scatter2d": "^3.1.3",
+    "regl-scatter2d": "git://github.com/gl-vis/regl-scatter2d.git#4dad59f2c2b71eece2931c71b8e3af7f87da2d7f",
```

Comment:
https://github.com/gl-vis/regl-scatter2d/compare/issue-3660

Reply:
Please make a PR to `regl-scatter2d` and ping @dy